packages/ERTP/src/types.js

```diff
@@ -101,9 +99,9 @@
  * as 'BTC' or 'moola') is provided by the maker of the issuer and should
  * not be trusted as accurate.
  *
- * Every amount created by AmountMath will have the same brand, but recipients
- * cannot use the brand by itself to verify that a purported amount is
- * authentic, since the brand can be reused by a misbehaving issuer.
+ * Every amount created by a particular AmountMath will share the same brand,
+ * but recipients cannot use the brand by itself to verify that a purported
+ * amount is authentic, since the brand can be reused by a misbehaving issuer.
```

Comment:
what does an `authentic amount` mean? I think this gets us in the bad use vs. mention territority. Any amount is authentic if you glue a brand and value together.

Reply:
reworded.